src/app/services/core-event-handlers.service.ts

```diff
@@ -89,12 +90,18 @@ export class CoreEventHandlersService {
     this.formSubmit.actions$
       .pipe(
       )
-      .subscribe(({ action }) => {
-        // console.log(`process action ${action}`);
+      .subscribe(({action, payload}) => {         
         if (action === 'refreshWorkflow') {
           this.workflow.refresh();
           return;
         }
+        /**
+         * Allow adapters to be installed by non-core flows?
+         */
+        if (action === 'installAdapter'){          
```

Comment:
Should we have a flag for this? E.g a debug flag?
Or maybe there is a good way to stub / substitute a flag or the core/non-core status?
@the-greenman

Reply:
I've now added a new settings flag to the core settings form.